packages/schematics/src/command-line/workspace-schematic.ts

```diff
@@ -115,7 +115,11 @@ function createWorkflow(dryRun: boolean) {
   const root = normalize(rootDirectory);
   const host = new virtualFs.ScopedHost(new NodeJsSyncHost(), root);
   return new NodeWorkflow(host, {
-    packageManager: fileExists('yarn.lock') ? 'yarn' : 'npm',
+    packageManager: fileExists('yarn.lock')
```

Comment:
Would this not be better looking at the angular.json or maybe nx.json file and getting the chosen package manager? My worried is we are still limiting this to just a few packages managers. At least this way we can dynamically add more managers without changing adding more if/switch statement

Reply:
You have to look at the global `angular.json` too